composer.json

```diff
@@ -35,6 +35,7 @@
         "aws/aws-sdk-php": "^2.4",
         "doctrine/cache": "^1.1",
         "doctrine/orm": "^2.3",
```

Comment:
I think this `require-dev` entry is useless. There isn't any code in this bundle which depends on `Doctrine\\ORM`!

We should remove it and keep `doctrine/persistence` as the true optional dependency thus we only provide an `AbstractDoctrineLoader` to help developers to implement their own doctrine ORM/ODM loader.